src/instant/views/AssetSelectorView.tsx

```diff
@@ -32,9 +33,11 @@ class AssetSelectorView extends React.Component<ViewProps> {
         <ul className={styles.list}>
           {
             Object.values(tokens).map((token, index) => {
+              const balance = user && user.account ? balances[token.symbol] : new BigNumber(0);
+
               return (
                 <li data-test-id={token.symbol.toLowerCase()} className={styles.listItem} key={index}>
-                  <Asset currency={token.symbol} balance={balances[token.symbol]} user={user}
+                  <Asset currency={token.symbol} balance={balance} user={user}
```

Comment:
wouldn't something ugly like `balances[token.symbol] || new BigNumber(0)` suffice?

Reply:
not really because the balances  doesn't exist so you are  trying to read a property from undefined :)